### DISCUSSION TITLE
Netbox Static Files directory

### DISCUSSION THREAD

**Discussion by EhtishamSabir** [netbox-community/netbox#11149] [Q&A]
I want to load index.html when I visit 
https://X.X.X.X/static/docs/
but it displays this 
Page not found (404)

Directory indexes are not allowed here.

It works fine when I visit https://X.X.X.X/static/docs/index.html

**Comment by mtinberg** [comment]
The static files are served by your webserver (nginx, apache, etc.) and I would guess the webserver needs to be configured to do the automatic redirect to index.html if it exists for that url location / directory


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: Ehtisham Sabir ***@***.***>
Sent: Friday, December 9, 2022 7:40 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Netbox Static Files directory (Discussion #11149)


I want to load index.html when I visit
https://X.X.X.X/static/docs/
but it displays this
Page not found (404)

Directory indexes are not allowed here.

It works fine when I visit https://X.X.X.X/static/docs/index.html
